tests/bzlmod/go_mod_test.bzl

```diff
@@ -117,8 +117,13 @@ go_mod_godebug_test = unittest.make(_go_mod_godebug_test_impl)
 
 _GO_SUM_CONTENT = """cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
 github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03qcyfWMU=
+<<<<<<<
```

Comment:
Probably not a big deal, but can we somehow escape this, so the IDE/git doesn't think that **this** file is not merged?